resources/definitions/jellybox.def.json

```diff
@@ -0,0 +1,35 @@
+        "file_formats": "text/x-gcode",
+        "has_materials": true,
+        "has_machine_materials": true
+    },
+
+    "overrides": {
+        "machine_name": { "default_value": "JellyBOX" },
+        "machine_width": { "default_value": 170 },
+        "machine_height": { "default_value": 145 },
+        "machine_depth": { "default_value": 160 },
+        "machine_nozzle_size": { "default_value": 0.4 },
+        "material_diameter": { "default_value": 1.75 },
+        "machine_heated_bed": { "default_value": true },
+        "machine_center_is_zero": { "default_value": false },
+        "machine_gcode_flavor": { "default_value": "RepRap (Marlin/Sprinter)" },
+        "machine_start_gcode": {
+            "default_value": "---------------------------------------\n; ; ; Jellybox Start Script Begin ; ; ;\n_______________________________________\n; M92 E140 ;optionally adjust steps per mm for your filament\n\n;                 Print Settings Summary\n; (overwriting these values will NOT change your printer's behavior)\n;              sliced for:  {machine_name}\n;         nozzle diameter:  {machine_nozzle_size}\n;       filament diameter:  {material_diameter}\n;            layer height:  {layer_height}\n;        1st layer height:  {layer_height_0}\n;              line width:  {line_width}\n;          wall thickness:  {wall_thickness}\n;          infill density:  {infill_sparse_density}\n;          infill pattern:  {infill_pattern}\n;       print temperature:  {material_print_temperature}\n;  heated bed temperature:  {material_bed_temperature}\n;       regular fan speed:  {cool_fan_speed_min}\n;           max fan speed:  {cool_fan_speed_max}\n;                 support?  {support_enable}\n;              spiralized?  {magic_spiralize}\n\nM117 Preparing        ;write Preparing\nM140 S{material_bed_temperature}              ;set bed temperature and move on\nM104 S{material_print_temperature}             ;set extruder temperature and move on\nM206 X10.0 Y0.0       ;set x homing offset for default bed leveling\nG21                   ;metric values\nG90                   ;absolute positioning\nM107                  ;start with the fan off\nM82                   ;set extruder to absolute mode\nG28                   ;home all axes\nM203 Z5               ;slow Z speed down for greater accuracy when probing\nG29                   ;auto bed leveling procedure\nM203 Z7               ;pick up z speed again for printing\nM190 S{material_bed_temperature}              ;wait for the bed to reach desired temperature\nM109 S{material_print_temperature}             ;wait for the extruder to reach desired temperature\nG92 E0                ;reset the extruder position\nG1 F200 E5            ;extrude 5mm of feed stock\nG92 E0                ;reset the extruder position again\nM117 Print starting   ;write Print starting\n---------------------------------------------\n; ; ; Jellybox Printer Start Script End ; ; ;\n_____________________________________________"
```

Comment:
tested and works. 
however, it's a good point - i'll comment it out to make sure. 

also, look how awesome it looks: 
![image](https://cloud.githubusercontent.com/assets/16858474/20274673/7b376740-aa63-11e6-81ec-a125dafe279a.png)